doc/index.rst

```diff
@@ -34,13 +33,9 @@ The PyPSA meets Africa team is currently running four types of meetings:
 
 - **Specific code meeting (by invitation)**
 
-  - meeting hosted on Discord
-  - join updates, demos, Q&A's, discussions and the coordination of each work package
-  
-    1. Demand creation and prediction meeting, every Wednesday 21:00 UTC+0, `download .ics file <hhttps://drive.google.com/file/d/1OBGBwSWeuYzhssb6tDxyAPIzz3uUhsnv/view?usp=sharing>`__
```

Comment:
I just noticed that there are some refuses when resolving the conflicts.
Please restore the previous version also here

Reply:
Fixed now (hopefully). Thank so much you for checking